static/app/views/replays/detail/console/index.tsx

```diff
@@ -1,19 +1,26 @@
 import styled from '@emotion/styled';
+import moment from 'moment';
 
 import {Panel} from 'sentry/components/panels';
 import {BreadcrumbTypeDefault} from 'sentry/types/breadcrumbs';
+import {getFormattedDate} from 'sentry/utils/dates';
 
 import ConsoleMessage from './consoleMessage';
 
 interface Props {
   breadcrumbs: BreadcrumbTypeDefault[];
+  startTimestamp: number;
 }
 
-function Console({breadcrumbs}: Props) {
+function Console({breadcrumbs, startTimestamp = 0}: Props) {
   return (
     <ConsoleTable>
       {breadcrumbs.map((breadcrumb, i) => (
         <ConsoleMessage
+          relativeTimestamp={getFormattedDate(
```

Comment:
Since the timestamp being passed thru didn't feel right I calculate the relative timestamp up here, but I can do within I just felt prop drilling two layers was a bit.